applications/yc_w16.md

```diff
@@ -0,0 +1,434 @@
+# YC Application
+
+hackEDU's nonprofit YC application for W16.
+
+### Application
+
+**Organization name**
+
+hackEDU
+
+**Organization url, if any**
+
+https://hackedu.us
+
+**If you have an online demo, what's the url?**
+
+N/A
+
+**What will your organization do?**
+
+hackEDU is the worldwide movement of free student-led coding clubs. We provide
+an open-source club playbook and a strong community of club leaders around the
+globe to high school students starting clubs. Our club playbook contains
+workshops that bring members from little or no experience to building things on
+their own with code and suggested facilitation guidelines for running a
+great club.
+
+**Where do you live now, and where would the organization be based after YC?**
+
+SF. SF.
+
+**Email address we should contact you at**
+
+zach@hackedu.us
+
+**Phone number(s)**
+
+[redacted]
+
+**Please provide an email address for each founder in the organization,
+including yourself. Founders must have at least 10% equity in the company. We
+will send an email to each founder to fill out additional information about
+themselves.**
+
+- zach@hackedu.us
+- jonathan@hackedu.us
+
+**Please enter the url of a 1 minute unlisted (not private) YouTube video
+introducing the founders.**
+
+TODO
+
+**Please tell us about an interesting project, preferably outside of class or
+work, that two or more of you created together. Include urls if possible.**
+
+We wrote a script that exploits referral codes to order food with delivery for
+less than $2 per meal. We used it nearly every weekday from November to June
+2014 to cut costs.
+
+**How long have the founders known one another and how did you meet? Have any
+of the founders not met in person?**
+
+We've known each other since September 2014 (13 months) and have been living
+together since November 2014 (11 months). We met after a mutual friend
+introduced Zach, who had just won PennApps, the University of Pennsylvania's
+hackathon, to Jonathan, who was a former lead organizer of PennApps.
+
+**How far along are you?**
+
+We're running an extended pilot with 47 schools across 11 states and 4
+countries, reaching 1500 students. 32% of our members are female (nearly 2x the
+national average) and 37% are minorities in tech (7x industry average).
+
+**If you've already started working on it, how long have you been working and
+how many lines of code (if applicable) have you written?**
+
+We've been working on hackEDU full-time for 11 months.
+
+- Lines of Markdown: 31,199 across 348 files (148,653 words)
```

Comment:
According to Amazon the average novel is 64,000 words long. Over two novels so far!